mockfactory/desi/redshift_smearing.py

```diff
@@ -27,44 +28,61 @@ def QSORedshiftSmearingRVS(fn=('data/qso_redshift_smearing_sv1.ecsv', 'data/qso_
             table = vstack([table[table['mean_z'] < tt['mean_z'][0]], tt])
         else:
             table = tt
-    rvs_laplace, rvs_gaussian, laz = [], [], []
+    rvs_nongaussian, rvs_gaussian, laz = [], [], []
     for iz, z in enumerate(table['mean_z']):
-        A0, x0, s0, sg, la = table['val_fit'][iz]
-        rvs_laplace.append(stats.laplace(x0, s0))
-        rvs_gaussian.append(stats.norm(x0, sg))
+        if tracer == 'QSO':
+            A0, x0, s0, sg, la = table['val_fit'][iz]
+            rvs_nongaussian.append(stats.laplace(x0, s0))
+            rvs_gaussian.append(stats.norm(x0, sg))
+        else:
+            sigma,x0, p,mu, la = table['val_fit'][iz]
+            rvs_nongaussian.append(stats.cauchy(scale=p/2, loc=mu))
+            rvs_gaussian.append(stats.norm(scale=sigma,loc=x0))
         laz.append(la)
     laz = np.array(laz)
     dztransform = lambda z, dz: dz / (constants.c / 1e3) / (1. + z)  # file units was km / s
-    return np.asarray(table['mean_z']), [rvs_laplace, rvs_gaussian], [laz, 1. - laz], dztransform
+    return np.asarray(table['mean_z']), [rvs_nongaussian, rvs_gaussian], [laz, 1. - laz], dztransform
 
 
-def QSORedshiftSmearing(fn=('data/qso_redshift_smearing_sv1.ecsv', 'data/qso_redshift_smearing_sv3.ecsv')):
+def tracerRedshiftSmearing(tracer='QSO',fn=('data/qso_redshift_smearing_sv1.ecsv', 'data/qso_redshift_smearing_sv3.ecsv')):
```

Comment:
I'd prefer TracerRedshiftSmearing (starting with a capital letter), as this function returns a class instance
'QSO',fn= => 'QSO', fn=